dali/python/nvidia/dali/auto_aug/augmentations.py

```diff
@@ -120,9 +120,10 @@ def contrast(data, parameter):
     It follows PIL implementation of Contrast enhancement which uses a channel-weighted
     mean as a contrast center.
     """
-    mean = fn.reductions.mean(data, axes=[0, 1])
+    # assumes FHWC or HWC layout
+    mean = fn.reductions.mean(data, axis_names="HW", keep_dims=True)
```

Comment:
I wonder if we should normalize over the each frame individually or over the whole sequence. The latter may result in a better temporal stability of the produced output, but I am not sure what kind of sequences we would need to work with, for this problem to actually appear.

Reply:
That's very good point. To sum up the off-line discussion, it seems to be done on per-frame basis in other frameworks. If needed we could include the other behaviour as an option, but for now I'd go with what is already in the wild to avoid surprises. For the record, this consideration applies also to auto_contrast and equalize that rely on some global statistics as well.